slasher/detection/service.go

```diff
@@ -88,9 +88,11 @@ func (ds *Service) Start() {
 	if featureconfig.Get().EnableHistoricalDetection {
 		// The detection service runs detection on all historical
 		// chain data since genesis.
-		go ds.detectHistoricalChainData(ds.ctx)
+		ds.detectHistoricalChainData(ds.ctx)
 	}
-
+	// We listen to a stream of blocks and attestations from the beacon node.
+	go ds.beaconClient.ReceiveBlocks(ds.ctx)
```

Comment:
Aren't these used in `beaconclient/service.go`?

Reply:
moved them here to start listening after historical detection ends

Reply:
Should they be removed then from historical detection?